### DISCUSSION TITLE
How to show html encoded content

### DISCUSSION THREAD

**Discussion by gaurikalp** [EnsembleUI/ensemble#115] [Q&A] [answered]
How to show html encoded content?

![HTMLEncodedContent](https://user-images.githubusercontent.com/115991454/203527946-b6fd4271-1aac-451f-ad56-f0771b94385e.PNG)

**Comment by vusters** [comment, marked as answer]
@gaurikalp You can use the Html widget and simply pass the html text in.

See Kitchen Sink: https://studio.ensembleui.com/app/e24402cb-75e2-404c-866c-29e6c3dd7992/screen/ZhTBody9YozadVvTlhW0

**Reply by gaurikalp** [reply]
Thanks you so much, Its working
